cmd/aries-js-worker/index.html

```diff
@@ -0,0 +1,32 @@
+<!--
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+<html>
```

Comment:
I assume this will become a test page where we run headless browser tests against?

Reply:
Yes, this is just a test page. I'll add a comment on it.

It is not being included in the build.